Models/Certificate.swift

```diff
@@ -249,14 +279,14 @@ private struct CertificateV1_1 : Certificate {
             let recipient = MethodsForV1_1.parse(recipientJSON: json["recipient"]),
             let assertion = MethodsForV1_1.parse(assertionJSON: json["assertion"]),
             let verifyData = MethodsForV1_1.parse(verifyJSON: json["verify"]),
-            let signature = json["signature"] else {
+            let signature = json["signature"] as? String? else {
```

Comment:
Nope, just `as? String`

If it can be nil, then it shouldn't be in the guard statement. You can just have 
```
self.signature = json["signature"] as? String
```
at the end of this method.